tests/page-objects/apdex/load.page.js

```diff
@@ -1,23 +1,30 @@
 const { $ } = require('@wdio/globals');
```

Comment:
Is this necessary? Can you please check if the test will pass if you remove this line?

Reply:
Removed thanks